### DISCUSSION TITLE
Does react-native-web run without errors in old Chromium or Webkit Engines?

### DISCUSSION THREAD

**Discussion by jeremies** [necolas/react-native-web#2484] [Q&A] [answered]
Which are the versions of Chromium or Webkit that react-native-web doesn't support? What could be the stoppers in the browser?

**Comment by necolas** [comment, marked as answer]
https://necolas.github.io/react-native-web/docs/browser-compatibility/

**Reply by jeremies** [reply]
Okay, thanks! 
What can I do if I want to support Chrome 30 for example?
